mindjunkies/dashboard/tests/test_views.py

```
@@ -1,114 +1,355 @@
 import pytest
+from django.contrib.messages import get_messages
 from django.urls import reverse
 from model_bakery import baker
+from django.core.files.uploadedfile import SimpleUploadedFile
 from django.contrib.auth.models import Permission
 from django.contrib.contenttypes.models import ContentType
 
 from mindjunkies.accounts.models import User
```

Ruff (0.8.2)

`mindjunkies.accounts.models.User` imported but unused

Remove unused import: `mindjunkies.accounts.models.User`

(F401)